go.mod

```diff
@@ -1,10 +1,11 @@
 module github.com/openconfig/featureprofiles
 
-go 1.17
+go 1.18
```

Comment:
Ondatra is still on go 1.17. Can we keep it on 1.17 for now?

Reply:
Yes, I verified and the test works with 1.17 as well